configuration/wireless/access-point-bridged.md

```diff
@@ -0,0 +1,157 @@
+
+Now enable the `systemd-networkd` service to create and populate the bridge when your Raspberry Pi boots:
+
+```
+sudo systemctl enable systemd-networkd
+```
+
+### Define the bridge device IP configuration
+
+Network interfaces that are members of a bridge device are never assigned an IP address, since they communicate via the bridge. The bridge device itself needs an IP address, so that you can reach your Raspberry Pi on the network.
+
+`dhcpcd`, the DHCP client on the Raspberry Pi, automatically requests an IP address for every active interface. So we need to block the `eth0` and `wlan0` interfaces from being processed, and let `dhcpcd` only configure `br0` via DHCP.
+
+```
+sudo nano /etc/dhcpcd.conf
+```
+
+Add the following line near the beginning of the file (above the first `interface xxx` line, if any):
+```
+denyinterfaces wlan0 eth0
+```
+Interface `br0` will be configured as per defaults via DHCP, no specific entry is necessary. Save the file to complete the IP configuration of the machine.
+
+<a name="ap-config"></a>
+## Configure the access point software
+
+Create the `hostapd` configuration file, located at `/etc/hostapd/hostapd.conf`, to add the various parameters for your wireless network. 
+
+```
+sudo nano /etc/hostapd/hostapd.conf
+```
+
+Add the information below to the configuration file. This configuration assumes we are using channel 7, with a network name of `NameOfNetwork`, and a password `AardvarkBadgerHedgehog`. Note that the name and password should **not** have quotes around them. The passphrase should be between 8 and 64 characters in length.
+
+To use the 5 GHz band, you can change the operations mode from `hw_mode=g` to `hw_mode=a`. Possible values for `hw_mode` are:
+ - a = IEEE 802.11a (5 GHz)
+ - b = IEEE 802.11b (2.4 GHz)
+ - g = IEEE 802.11g (2.4 GHz)
+ - ad = IEEE 802.11ad (60 GHz)
+
+```
+interface=wlan0
+bridge=br0
+ssid=NameOfNetwork
+hw_mode=g
+channel=7
+macaddr_acl=0
+auth_algs=1
+ignore_broadcast_ssid=0
+wpa=2
+wpa_passphrase=AardvarkBadgerHedgehog
+wpa_key_mgmt=WPA-PSK
+wpa_pairwise=TKIP
+rsn_pairwise=CCMP
+```
+Note the lines `interface=wlan0` and `bridge=br0`: these direct `hostapd` to add the `wlan0` interface as bridge member to `br0` when the access point starts, completing the bridge between ethernet and wireless.
+
+<a name="conclusion"></a>
+## Run your new wireless access point
+
+It is now time restart your Raspberry Pi and verify the wireless access point becomes automatically available.
+
+```
+sudo systemctl reboot
+```
+Once your Raspberry Pi has restarted, search for WiFi networks with your wireless client. The network SSID you specified in file `/etc/hostapd/hostapd.conf` should now be present, and it should be accessible with the specified password.
+
+If your wireless client has access to the local network and the Internet, congratulations on your new access point!
```

Comment:
"congratulations on setting up your new access point!" ?

Reply:
Is it obviously frenglish? Anyway do as you see best